docs/push-flow.md

```diff
@@ -0,0 +1,21 @@
+■ data push-flow [PATH]
+
+  `PATH` (optional) is the path to the data package.
```

Comment:
Probably we should use "dataset" instead of "data package"

Reply:
@anuveyatsu this is grabbed from `data push` docs end modified. So if we do we should change for it as well